content/docs/pipelines/overview/pipelines-overview.md

```diff
@@ -235,31 +236,29 @@ At a high level, the execution of a pipeline proceeds as follows:
 * **Pipeline Service**: You call the Pipeline Service to create a
   pipeline run from the static configuration. 
 * **Kubernetes resources**: The Pipeline Service calls the Kubernetes API
-  server to create the necessary Kubernetes resources
-  ([CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/))
-  to run the pipeline.   
+  server. The API server creates the Kubernetes resources
```

Comment:
I would recommend keeping the original version of the "Kubernetes resources" section. Its a bit shorter, and I think it more correct to say that the API calls to the Kubernetes API create resources required by the pipeline run.

Reply:
👍